### ISSUE TITLE
Support for the PaLM API in Java

### ISSUE THREAD

**Issue by glaforge** [microsoft/semantic-kernel#2891]
It would be great if Semantic Kernel supported the PaLM API from Google in Java.
There's the developer focused one [here](https://developers.generativeai.google/products/palm) or ideally the Cloud-based one from Vertex AI [here](https://cloud.google.com/vertex-ai/docs/generative-ai/learn/overview).

There are different LLM models available (chat and non-chat), and you can also compute embeddings.

**Comment by nacharya1** [comment]
@glaforge we recently add PaLM support apart of #2258 and #2076. Hopefully this helps.

**Comment by glaforge** [comment]
Sorry, I should have been clearer.
It was following a discussion with @brunoborges about the support of PaLM, but in the Java flavor of Semantic Kernel.
Could we reopen this issue?

**Comment by nacharya1** [comment]
yes course. Appreciate the clarity @glaforge. Thanks for reopening Bruno.

**Comment by brunoborges** [comment]
Definitely one of the downsides of having all languages in the same repository is that issues may be opened related to one language but reviewed by another language maintainer.

But there are upsides! Let's help users being clarity as they open issues in the future.
